src/check.h.in

```diff
@@ -170,6 +170,23 @@ CK_DLL_EXP void CK_EXPORT suite_add_tcase(Suite * s, TCase * tc);
 CK_DLL_EXP TCase *CK_EXPORT tcase_create(const char *name);
 
 /**
+ * Create a test case associated with certain tags.
+ *
+ * Once created, tests can be added with the tcase_add_test()
+ * function, and the test case assigned to a suite with the
+ * suite_add_tcase() function.
+ *
+ * @param name name of the test case
+ * @param tags string containing arbitrary tags separated by spaces
+ *
+ * @return test case containing no tests
+ *
+ * @since 0.10.0
+ * */
+CK_DLL_EXP TCase *CK_EXPORT tcase_create_tagged(const char *name,
```

Comment:
All test cases can have tags, it is just that the default state is to have a NULL list of tags, right? Why not add a method to add a tag to a test case?

As an example, every test case can have a timeout. This is applied using
```
tcase_set_timeout(TCase * tc, double timeout);
```
Perhaps it makes more sense to have a tagging function:
```
tcase_add_tag(TCase * tc, char* tag);
```

Is the use case such that adding multiple tags as once is desired? Should this really be:
```
tcase_add_tags(TCase * tc, char* tags);
```
or
```
tcase_add_tags(TCase * tc, char** tags);
```
? You are assuming that tags are space delimited. By convention this is true everywhere I have seen, but I attempted to find a "formal" definition for this and failed. Maybe it makes sense to have tags space delimited. I cannot think of a case where it does not make sense.